sdk/keyvault/test-resources.json

```diff
@@ -67,24 +67,24 @@
     },
     "keyVaultSku": {
       "type": "string",
-      "defaultValue": "Premium",
+      "defaultValue": "premium",
       "allowedValues": [
-        "Standard",
-        "Premium"
+        "standard",
+        "premium"
       ],
       "metadata": {
         "description": "Key Vault SKU to deploy. The default is 'Premium'"
       }
     },
-    "endpointSuffix": {
-      "type": "String",
-      "defaultValue": "vault.azure.net"
+    "keyVaultEndpointSuffix": {
```

Comment:
> Match with the ArmTemplateParameter setting, so change the parameter name of the endpoint suffix to keyVaultEndpointSuffix.

Could you please where to find document about `ArmTemplateParameter`?

Reply:
Please refer to is[sue#23646 ](https://github.com/Azure/azure-sdk-for-java/issues/23646)